PythonVisualizations/Array.py

```diff
@@ -197,19 +219,25 @@ def createCellValue(self, indexOrCoords, key, color=None):
         return cell_rect, cell_val
 
     def display(self):
+        callEnviron = self.createCallEnvironment()

```

Comment:
The `display()` method sets up the basic display elements including the cells of the array.  It isn't one of the calls that requires animation and does not need a call environment.  You can get rid of the call to `createCallEnvironment()` here.

Reply:
Is there a reason you kept the call to `self.createCallEnvironment()` here in the `display()` method?  I think lines 217 and 234 (the call to `cleanUp(callEnviron)` should be removed.